snippets/post_install_network_config

```diff
@@ -77,6 +77,9 @@ mv /etc/sysconfig/network.cobbler /etc/sysconfig/network
 # Also set the hostname now, some applications require it
 # (e.g.: if we're connecting to Puppet before a reboot).
 /bin/hostname $hostname
+        #if $osversion == "rhel7"
```

Comment:
There is probably a better check because it's also the case on many distro using systemd.

Instead I would test if the hostnamectl command is present (that is part of systemd), and use /usr/bin/hostnamectl set-hostname $hostname  , fall back to hostname else.